doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -727,6 +727,30 @@ declare your constant as rigid for proof search using the command
 Strategies for rewriting
 ------------------------
 
+Usage
+~~~~~
+
+.. tacn:: rewrite_strat @rewstrategy {? in @ident }
+   :name: rewrite_strat
+
+   Rewrite using the strategy s in hypothesis ident or the conclusion.
+
+   .. exn:: Nothing to rewrite.
+
+      If the strategy failed.
+
+   .. exn:: No progress made.
+
+      If the strategy succeeded but made no progress.
+
+   .. exn:: Unable to satisfy the rewriting constraints.
+
+      If the strategy succeeded and made progress but the
+      corresponding rewriting constraints are not satisfied.
+
+   The ``setoid_rewrite c`` tactic is basically equivalent to
+   ``rewrite_strat (outermost c)``.
```

Comment:
Do you want to comment on the differences?  @mattam82 can you explain exactly what the differences are?

Reply:
No.  Let's not leverage responding to your initial comments into rewriting the whole thing.

Reply:
Ah, okay.  Another question here.  Why the current thing rather than
```suggestion
   The `setoid_rewrite c` tactic is basically equivalent to
   `rewrite_strat (outermost c)`.
```
or
```suggestion
   The :g:`setoid_rewrite c` tactic is basically equivalent to
   :g:`rewrite_strat (outermost c)`.
```
or
```suggestion
   The :n:`setoid_rewrite @one_term` tactic is basically equivalent to
   :n:`rewrite_strat (outermost @one_term)`.
```

Reply:
Both ``` ` ``` and ``` `` ``` give nonbold monospaced text,  ``` :n:` ``` enables regex graphics (as in the tactic definitions--boxes and hyperlinks to nonterminals with ``` @nonterminal ```).  ``` :g:` ``` seems to be the same as ``` ` ```, I haven't looked into it.

![image](https://user-images.githubusercontent.com/1253341/103579634-7b3eca80-4e8d-11eb-8e93-879fb9b4236a.png)

Unfortunately, `:n:` doesn't yet handle hyperlinks to tactics and commands, so I'll do it this way:

```
   :tacn:`setoid_rewrite` :n:`@one_term` is basically equivalent to
   :n:`rewrite_strat outermost @one_term`.
```

![image](https://user-images.githubusercontent.com/1253341/103581415-d6be8780-4e90-11eb-8ebc-5c01fb695e41.png)